e2e/react-start/rsc-hmr/tests/hydration.ts

```
@@ -0,0 +1,7 @@
+import { expect, type Page } from '@playwright/test'
```

ESLint

[error] Member 'Page' of the import declaration should be sorted alphabetically.

(sort-imports)